wolfcrypt/src/port/Renesas/renesas_tsip_util.c

```diff
@@ -55,6 +55,27 @@ static uint32_t     g_encrypted_publicCA_key[R_TSIP_SINST_WORD_SIZE];
 static uint32_t     g_CAscm_Idx;          /* index of CM table    */
 #endif
 
+/* for benchmark test */
```

Comment:
For code that is only used in the benchmark or wolfCrypt test, can we move that into the ./IDE directory, or benchmark.c?  I don't think we should put benchmark-only code in the library proper (./wolfcrypt/src/).

Reply:
Moved all the functions used only in Benchmark to benchmark.c.